### ISSUE TITLE
Reference rich-text fields should allow PASTE

### ISSUE THREAD

**Issue by lpanebr** [substance/texture#449]
Paste CTRL+V does not work in fields that allow bold, italic etc.

Tested with;
 - Article Title
 - Book Title

**Comment by michael** [comment]
Yeah these HTML editors have some issues which we need to address, in a dedicated effort. We can work on this in April/May.

Meanwhile, would it be ok to make all fields plain text inputs?

**Comment by lpanebr** [comment]
I think so. Retyping titles is very painful.. But you probably want to hear from Fabio and Melisa about this.

**Comment by Melissa37** [comment]
I agree - retyping is the absolute worst and people will think the system is awful if they cannot copy and paste into it! It's fine if formatting is lost to start with.
